tests/integration/servicecatalog/cmd_service_test.go

```diff
@@ -279,4 +279,153 @@ var _ = Describe("odo service command tests", func() {
 			helper.CmdShouldPass("odo", "service", "delete", "-f", "dh-postgresql-apb", "--app", app, "--project", project)
 		})
 	})
+
+	Context("When link backend between component and service", func() {
+		JustBeforeEach(func() {
+			context = helper.CreateNewContext()
+			os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+			project = helper.CreateRandProject()
+			context1 = helper.CreateNewContext()
+			context2 = helper.CreateNewContext()
```

Comment:
These lines are common in test spec that you updated form link test file. You can create a separate function that only does above operation and call the function here.

Reply:
@prietyc123 Use something like this 
```
setup := func() {
		project = helper.CreateRandProject()
		context1 = helper.CreateNewContext()
		context2 = helper.CreateNewContext()
	}

	removeSetup := func() {
		helper.DeleteProject(project)
		helper.DeleteDir(context)
		helper.DeleteDir(context1)
		helper.DeleteDir(context2)
	}
```
And call these in the required place